Examples/CleanseGithubBrowser/CleanseGithubBrowserTests/CleanseGithubBrowserTests.swift

```diff
@@ -14,10 +14,13 @@ import Cleanse
 class CleanseGithubBrowserTests: XCTestCase {
     func testMembersViewController_Title() {
         // Tests that the member's view controller sets the title appropriately.
+        let factory = Factory<AssistedInjectionTags.MembersPageSettingsSeed> { _ in
```

Comment:
Interesting. I am wondering how do you deal with the `seed`

Reply:
Yep it is! For the purposes of this test case I don't care about the seed.